content/blog/2020-to-2021.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "2020 -> 2021"
+description = "Reflecting and looking forward."
+date = 2021-01-21
+[extra]
+author = "Josh Lehman"
+ship = "~wolref-podlex"
+image = "https://media.urbit.org/site/posts/essays/arch-sunrise.jpeg"
++++
+
+<br>
+
+![arch](https://media.urbit.org/site/posts/essays/arch-sunrise.jpeg)
+
+<br>
+
+In this post, which has been adapted from our most recent newsletter, we'll take a moment to reflect on the events of the last year, and then turn our attention to the next one.
```

Comment:
Can we link directly to the newsletter signup? Anyone who shows up here off the web might want to sign up.

Reply:
Is there a dedicated place to sign up for the newsletter that isn't the urbit.org landing page? Otherwise can just link to that.

Reply:
I think @matildepark or @urcades might know. I think you can link directly to it, and we might have a component you can drop in?

Reply:
There is a gobbledygook URL from mailchimp here: 
https://urbit.us11.list-manage.com/subscribe/post?u=972a03db9e0c6c25bb58de8c8&amp;amp;id=be143888d2

It's good to note that all blog post entries have the newsletter signup form underneath them:

<img width="816" alt="Screen Shot 2021-01-25 at 5 07 56 PM" src="https://user-images.githubusercontent.com/1195363/105772244-e1b88500-5f2f-11eb-8f98-46ac712d32f8.png">

Reply:
Does that sign people up for Urbit or Tlon? Probably just Urbit

Reply:
@urcades The gobbledygook URL no longer appears to work unfortunately.

@matildepark Pretty sure this just goes to the Urbit newsletter. Instead of linking then I'll probably just say "scroll to the bottom of the page to sign up for our newsletter" or something to that effect.

Reply:
[This one does?](https://urbit.us11.list-manage.com/subscribe/post?u=972a03db9e0c6c25bb58de8c8&amp;id=be143888d2)

Reply:
Oh, dope.